include/nuttx/net/netdev.h

```diff
@@ -1,16 +1,10 @@
 /****************************************************************************
  * include/nuttx/net/netdev.h
- * Defines architecture-specific device driver interfaces to the NuttX
- * network.
  *
- *   Copyright (C) 2007, 2009, 2011-2018 Gregory Nutt. All rights reserved.
- *   Author: Gregory Nutt <gnutt@nuttx.org>
- *
- * Derived largely from portions of uIP with has a similar BSD-styple
- * license:
- *
- *   Copyright (c) 2001-2003, Adam Dunkels.
- *   All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * SPDX-FileCopyrightText: 2007, 2009, 2011-2018 Gregory Nutt. All rights reserved.
```

Comment:
Hi @jerpelea , I found that our `checkpatch.sh`(`nxstyle.c`) does not allow this long line, but I don't know which one should we modify, this header file or `nxstyle.c`?
```bash
./tools/checkpatch.sh -f include/nuttx/net/netdev.h
include/nuttx/net/netdev.h:5:83: error: Long line found
```

Reply:
it is a long line nxtools is right